src/PeopleSDKAdapter.js

```diff
@@ -184,4 +185,27 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
 
     return this.getPersonObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits people list
+   *
+   * @param {string} query search query
+   * @returns {external:Observable.<Person[]>} Observable that emits person list based on search query
+   */
+  getPeopleList(query) {
+    logger.debug('PEOPLE', query, 'getPeopleList()', ['called with', {query}]);
```

Comment:
the second param to the logger functions is the entity id (for example, a person id). As we do not have a person id here, you can use 'undefined' or 'null'.